README.md

```diff
@@ -34,10 +34,10 @@ $ ./build.sh --no-cache wdqs
 ### Test Commands
 
 ```
-$ make test
-$ make test SUITE=repo
-$ make test SUITE=repo FILTER=special-item
-$ make test-upgrade VERSION=wmde.9
+$ ./test.sh
+$ ./test.sh repo
+$ ./test.sh repo --spec special-item
```

Comment:
Please update to note the need for path, extension, and the root of the path

Reply:
As a follow-up, we might want to see if there's a way to default that to specs/SUITE/ as the root?

Reply:
Why using a search string (i.e. "property") vs the path to the spec didn't work is actually a known issue on WDIO 8 which was resolved in August, but we're not currently up enough minor/patch version to get it. Here is the related Github issue: https://github.com/webdriverio/webdriverio/issues/9495

It's equally of issue in our current test system using the `FILTER=` env var. Does `make test SUITE=repo FILTER=property` work for you on main either? It doesn't currently for me. 

Otherwise, I actually want the file path to be explicit so that everything in test always has the same file root for simplicity. These are the same paths we us in the `conf.ts` files for pointing to spec files. Both settings are direct passthroughs to WDIO so they work the same and the WDIO docs are valid. That all makes for me less cognitive dissonance. Also this is modeled in such a way thar any spec/s can be ran against any test env setup and the two aren't unnecessarily coupled.